DirectProgramming/DPC++FPGA/Tutorials/Tools/experimental/platform-designer/add-quartus/.gitignore

```diff
@@ -0,0 +1,29 @@
+*
```

Comment:
Please remove this file

Reply:
I'll remove before i submit; I like this because I don't have to delete my build directory every time I commit. furthermore: Intel® Quartus® Prime does in-source builds so it makes a mess